test/hotspot/jtreg/serviceability/jvmti/GetOwnedMonitorInfo/GetOwnedMonitorInfoTest.java

```diff
@@ -46,19 +46,40 @@ public class GetOwnedMonitorInfoTest {
         }
     }
 
+    private static native void jniMonitorEnter(Object obj);
     private static native int check();
     private static native boolean hasEventPosted();
 
+    private static void jniMonitorEnterAndLetObjectDie() {
+        // The monitor iterator used by GetOwnedMonitorInfo used to
+        // assert when an owned monitor with a dead object was found.
+        // Inject this situation into this test that performs other
+        // GetOwnedMonitorInfo testing.
+        Object obj = new Object() { public String toString() {return "";} };
```

Comment:
Nit: the `toString` definition is not needed. This could just be `new Object();`, or `new Object() {};` if you want to introduce a nested class.

Reply:
Thanks. Done.